### PR TITLE
jewel: rgw: multisite: realm rename does not propagate to other clusters

### PR DESCRIPTION
**Pull request by smithfarm**
http://tracker.ceph.com/issues/19775

### CODE REVIEW

**Comment by smithfarm** [comment]
Jenkins re-test this please (unittest_throttle)

File: src/rgw/rgw_admin.cc [hunk 1]
```diff
@@ -2993,6 +2993,9 @@ int main(int argc, char **argv)
 	  cerr << "realm.rename failed: " << cpp_strerror(-ret) << std::endl;
 	  return -ret;
 	}
+        cout << "Realm name updated. Note that this change only applies to "
+            "the current cluster, so this command must be run separately "
```
Comment (amitkumar50) [inline src/rgw/rgw_admin.cc]: @smithfarm I believe this is incorrect sentence"so this command must be run separately" should be something as:
"so this command must run separately"

Comment (smithfarm) [inline src/rgw/rgw_admin.cc]: @amitkumar50 Backports are cherry-picked from master, so any changes would have to go into master, first.

("this command must be run separately" is fine as it is, though)

**Comment by smithfarm** [comment]
@oritwas This passed an rgw suite at http://tracker.ceph.com/issues/20613#note-18

Please review.

**Pull request merged**